### PR TITLE
Upgrade to spark 3.0.1

### PR DESCRIPTION
**Pull request by ilubis-pub**
Upgrade to Spark SQL 3.0.1

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -5,7 +5,7 @@ organization := "org.zalando"
 
 scalaVersion := "2.12.10"
 
-libraryDependencies += "org.apache.spark" %% "spark-sql" % "2.4.7"  % Provided
+libraryDependencies += "org.apache.spark" %% "spark-sql" % "3.0.1"  % Provided
 libraryDependencies += "com.typesafe.play" %% "play-json" % "2.9.1"
 dependencyOverrides ++= Set("com.fasterxml.jackson.core" % "jackson-databind" % "2.6.5")
```
Comment (hesserp) [inline build.sbt:10]: for me the tests did not work unless also upgrading `jackson-databind`. Version 2.10.5 seemed to work, while it complained for higher versions.

**Comment by hesserp** [comment]
:+1:

**Comment by ilubis-pub** [comment]
👍

**Comment by ilubis-pub** [comment]
👍

**Pull request merged**